client/src/components/GoogleMap/GoogleMap.js

```diff
@@ -0,0 +1,68 @@
+import React, { Component } from 'react';
+import { GoogleApiWrapper, InfoWindow, Map, Marker } from 'google-maps-react';
+class GoogleMap extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      showingInfoWindow: false,
+      activeMarker: {},
+      selectedPlace: {},
+    };
+    // binding this to event-handler functions
+    this.onMarkerClick = this.onMarkerClick.bind(this);
```

Comment:
@taleymirza we dot not use this convention anymore, if we have arrow functions then we dont need this

Reply:
Yup, I use this code from a resource to test map, I will refactor it